src/main/java/com/imsweb/mph/DefaultHematoDbUtilsProvider.java

```diff
@@ -123,4 +127,18 @@ else if (_transformFromDto.containsKey(leftCode)) {
         }
         return false;
     }
+
+    @Override
+    public Date getDataLastUpdated() {
+
+        File lastUpdatedDateFile = new File(System.getProperty("user.dir").replace(".idea\\modules", "") + "/src/main/resources/hemato_last_update_date.txt");
```

Comment:
You cannot get an internal resource this way, it won't work in a JAR file. This trick can be used in unit tests when the test needs to get a physical file (and even that should be rare), but you just can't do that in "read" code. You need to use a "Thread.currentThread.getResourcesAsStreamt" etc... In a try/catch with resources ideally.